test/ea_restaurant_data_loader_web/controllers/oauth2_controller_test.exs

```diff
@@ -49,5 +53,178 @@ defmodule EaRestaurantDataLoader.Test.EaRestaurantDataLoaderWeb.Controllers.Oaut
 
       assert_raise(BadRequest, ~r/bad request/, fn -> post(conn, "/login") end)
     end
+
+    test "refresh token - refresh unexpired access token", %{conn: conn} do
```

Comment:
The description of the test should start saying `test "should ...."`, i.e. `test "should not refresh a valid token"`

Reply:
Done.